requirements.txt

```diff
@@ -1,36 +1,49 @@
+#
+# These requirements were autogenerated by pipenv
+# To regenerate from the project's Pipfile, run:
+#
+#    pipenv lock --requirements
+#
+
 -i https://pypi.org/simple
-aiofiles==0.6.0
-asyncpg==0.21.0
+aiofiles==0.7.0
+asyncpg==0.23.0
 bcrypt==3.2.0
-cffi==1.14.4
+cffi==1.14.5
 click==7.1.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
-databases==0.4.1
-dnspython==2.0.0; python_version >= '3.6'
+databases==0.4.3
+deprecated==1.2.12; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+dnspython==2.1.0; python_version >= '3.6'
 email-validator==1.1.2
-environs==9.2.0
-fastapi-users==3.1.1
-fastapi==0.62.0
+environs==9.3.2
+fastapi-users==6.1.0
+fastapi==0.65.1
 flask-autoindex==0.6.6
 flask-silk==0.2
-flask==1.1.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
+flask==2.0.1; python_version >= '3.6'
 future==0.18.2; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2, 3.3'
-h11==0.11.0
-idna==2.10; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-itsdangerous==1.1.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-jinja2==2.11.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
-makefun==1.9.5
-markupsafe==1.1.1; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-marshmallow==3.9.1; python_version >= '3.5'
+h11==0.12.0; python_version >= '3.6'
+httptools==0.2.0
+idna==3.1; python_version >= '3.4'
+itsdangerous==2.0.1; python_version >= '3.6'
+jinja2==3.0.1; python_version >= '3.6'
+makefun==1.11.3
+markupsafe==2.0.1; python_version >= '3.6'
+marshmallow==3.12.1; python_full_version >= '3.5.0'
 passlib[bcrypt]==1.7.4
 psycopg2==2.8.6
 pycparser==2.20; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-pydantic==1.7.3; python_version >= '3.6'
-pyjwt==1.7.1
-python-dotenv==0.15.0
+pydantic==1.8.2; python_full_version >= '3.6.1'
+pyjwt==2.1.0; python_version >= '3.6'
+python-dotenv==0.17.1
 python-multipart==0.0.5
-six==1.15.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-sqlalchemy==1.3.20; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-starlette==0.13.6; python_version >= '3.6'
-tenacity==6.3.1
-uvicorn==0.13.1
-werkzeug==1.0.1; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
+six==1.16.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+sqlalchemy==1.3.24; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+starlette==0.14.2; python_version >= '3.6'
+tenacity==7.0.0
+typing-extensions==3.10.0.0
+uvicorn==0.13.4
+uvloop==0.15.2
+watchgod==0.7
```

Comment:
are all these requirements needed ?

Reply:
Yes, except `watchgod` should be a development dependency. 

`uvloop` and `httptools`, as suggested by [uvicorn's docs](https://www.uvicorn.org/#quickstart) should bring us performance improvements in serving concurrent requests.